docs/plugins/filters/jdbc_streaming.asciidoc

```diff
@@ -5,9 +5,9 @@
 ///////////////////////////////////////////
 START - GENERATED VARIABLES, DO NOT EDIT!
 ///////////////////////////////////////////
-:version: v1.0.7
-:release_date: 2019-05-30
-:changelog_url: https://github.com/logstash-plugins/logstash-filter-jdbc_streaming/blob/v1.0.7/CHANGELOG.md
+:version: v1.0.6
+:release_date: 2019-05-01
+:changelog_url: https://github.com/logstash-plugins/logstash-filter-jdbc_streaming/blob/v1.0.6/CHANGELOG.md
 :include_path: ../../../../logstash/docs/include
```

Comment:
Regression from lockfile. These changes are nice, but probably not critical (spelling, typos, etc.)